apps_product_creator/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Github product creator',
+    'summary': 'Create a product variant (by version) for every module found.',
```

Comment:
out of context, this description is cryptic...
I would suggest something like:
"Create a product variant (by version) for every module found in the Github repositories"